<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportSetDialog</class>
 <widget class="QDialog" name="ReportSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报告内容与格式</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>451</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>241</width>
     <height>182</height>
    </rect>
   </property>
   <property name="title">
    <string>变电站信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>报告标题：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lne_ReportTitle"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>变电站名称：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lne_TransStationName"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>变电站地址：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lne_TransStationAddress"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>回路名称：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lne_ReturnCircuitName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>回路地址：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lne_ReturnCircuitAddress"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>测试人员：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lne_TestPersionName"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>201</y>
     <width>241</width>
     <height>208</height>
    </rect>
   </property>
   <property name="title">
    <string>被测装置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>装置名称：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lne_DeviceTestedName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>装置型号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lne_DeviceTestedID"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>装置地址：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lne_DeviceTestedAddress"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>生产厂家：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lne_Manufacturer"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>序列号：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lne_SerialNumber"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>定制单编号：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lne_CustomOrderID"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>附加信息：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lne_AddInfo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="pbn_Ok">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbn_Cancle">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
